.gitpod.yml

```diff
@@ -0,0 +1,3 @@
+tasks:
+  - before: sudo wget -qO /usr/local/bin/yq https://github.com/mikefarah/yq/releases/latest/download/yq_linux_amd64 && sudo chmod +x /usr/local/bin/yq
+  - before: sudo ./experimental_docker_compose_files/02_custom_docker_file_connecting_agent_and_controller/gitpodURL.sh
```

Comment:
```suggestion
  - before: ./experimental_docker_compose_files/02_custom_docker_file_connecting_agent_and_controller/gitpodURL.sh
```

Reply:
I am getting this error 
```
gitpod /workspace/GSoC-2023-docker-based-quickstart (bug/empty_url_issue34) $  HISTFILE=/workspace/.gitpod/cmd-1 history -r; {
> sudo ./experimental_docker_compose_files/02_custom_docker_file_connecting_agent_and_controller/gitpodURL.sh
> }
Jenkins can Be accessed here: https://8080-
./experimental_docker_compose_files/02_custom_docker_file_connecting_agent_and_controller/gitpodURL.sh: line 13: /usr/local/bin/yq: Permission denied
```
even when permissions are set to `-rwxr-xr-x 1 root root 8941568 May 23 03:40 /usr/local/bin/yq`
I've tried `init` instead of `before` but same error, 
and it turns out the variables are also not set  when `before` command is running I used $GITPOD_REPO_ROOT to define path  in config file but it was giving error which got solved from using absolute path